Makefile

```diff
@@ -186,8 +186,11 @@ htmls:
 		mkdir -p html && $(PANDOC) $$f -o html/`basename $$f`.html; \
 	done
 
-doc/pgbouncer.1 doc/pgbouncer.5:
-	$(MAKE) -C doc $(@F)
+doc/pgbouncer.1:
+	$(MAKE) -C doc pgbouncer.1
+
+doc/pgbouncer.5:
+	$(MAKE) -C doc pgbouncer.5
```

Comment:
Is this change really necessary to fix the issue? Or is the `rm` enough.

Reply:
We do a lot of nightly runs and this fix in our internal repository allowed us to eliminate the time spent on false build errors. I consider it necessary to share with the community

Reply:
as far as I can tell this part of the change achieves nothing. It's simply a more verbose (but arguably clearer) syntax for the same thing.